layer4/evaluation_plan_test.go

```diff
@@ -0,0 +1,267 @@
+package layer4
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_ToMarkdownChecklist(t *testing.T) {
+	tests := []struct {
+		name           string
+		evaluationPlan EvaluationPlan
+		contains       []string
+		notContains    []string
+	}{
+		{
+			name: "comprehensive evaluation plan with multiple controls and assessments",
+			evaluationPlan: EvaluationPlan{
+				Plans: []AssessmentPlan{
+					{
+						Control: Mapping{
+							ReferenceId: "NIST-800-53",
```

Comment:
This is minor, but I think 800-53 would align closer with Layer 1 so this test data could add a bit of confusion.